lib/Doctrine/Common/Annotations/CachedReader.php

```diff
@@ -230,6 +230,12 @@ private function isCacheFresh($cacheKey, ReflectionClass $class)
     private function getLastModification(ReflectionClass $class)
     {
         $filename = $class->getFileName();
+        $cacheKey = '[T]'.$filename;
+
+        if (isset($this->loadedAnnotations[$cacheKey])) {
```

Comment:
I'd use a separate array for this, and type it with `int[]`, that way there will be no need to prefix it and SA tools will be happy.

Reply:
@greg0ire I suggest to name it `loadedFilemtimes`. My issue is to purge it, can I reset it in the method names `clearLoadedAnnotations`. Seems counter productive, but I do not see it feasable to add another method to clear things, and it will not be backward compatible.

Reply:
It sounds reasonable to clear it in the existing method to me